examples/rosettacode/rock_paper_scissor.nit

```diff
@@ -0,0 +1,50 @@
+# This file is part of NIT ( http://www.nitlanguage.org ).
+# This program is public domain
+#
+# Task: Programming_Task/Rock-paper-scissors
+# SEE: <http://rosettacode.org/wiki/Rock-paper-scissors>
+#
+# author: Istvan szalai 22/05/2015
+
+#return a value to dertermine who win the match
```

Comment:
Comments on functions should be formatted this way:

* There is a space between the `#` and the start of the comment
* There is an uppercase at the start of the line
* There is (optionally) a dot at the end of the line

Example:

~~~nit
# This is a nice and tidy Nit commend.
~~~